asdf/tagged.py

```diff
@@ -35,9 +35,9 @@
 
 import six
 
+from astropy import time
```

Comment:
On May 18, 2016, at 2:55 PM, Nadia Dencheva <notifications@github.com> wrote:

> In asdf/tagged.py:
> 
> > @@ -35,9 +35,9 @@
> >  
> >  import six
> >  
> > +from astropy import time
> 
> I haven't looked at the PR but could you place this import statement somewhere in a function or class so that it's not always required.

The import needs to be at the top level because class TaggedTime subclasses it and it is defined at the top level.